code/modules/power/lighting.dm

```diff
@@ -2,12 +2,12 @@
 //
 // consists of light fixtures (/obj/machinery/light) and light tube/bulb items (/obj/item/weapon/light)
 
-// status values shared between lighting fixtures and items
-#define LIGHT_OK     0
+// status values shared between lighting fixtures and items MOVED TO _DEFINES/lighting.DM
```

Comment:
Don't keep this, move any comments to lighting.dm and remove what you've commented out.